mep/people/templates/people/membership_activities.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static %}
+
+{% block page-subtitle %}{{ member.name }} · Library Members · {% endblock %}
+
+{% block js %}
+<script type="application/ld+json">{{ page_jsonld|safe }}</script>
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.name style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+<aside>
```

Comment:
I agree it feels weird as an aside. The markup didn't work properly without a container and I wasn't sure the right way to resolve. It would be nice to not have it wrapped in anything.

Where's the right place to customize the styles for the breadcrumbs to handle the differences for displaying properly on these pages?